client/src/graphics/animated_sprite.lua

```diff
@@ -100,13 +95,9 @@ function AnimatedSprite:update()
     end
   end
 end
-
-  function AnimatedSprite:draw(x, y, rot, x_scale, y_scale)
-    GraphicsUtil.drawBatch(self.animations[self.currentAnim].spriteSheet, self.animations[self.currentAnim].frames[self.currentFrame][1], x, y, rot, x_scale, y_scale)
-  end
   
-  function AnimatedSprite:qdraw(x, y, rot, x_scale, y_scale)
-    GraphicsUtil.drawQuad(self.animations[self.currentAnim].image, self.animations[self.currentAnim].frames[self.currentFrame][1], x, y, rot, x_scale, y_scale)
+  function AnimatedSprite:draw(x, y, rot, x_scale, y_scale)
+    GraphicsUtil.drawQuad(self.animations[self.currentAnim].image, self.animations[self.currentAnim].frames[self.currentFrame].quad, x, y, rot, x_scale, y_scale)
```

Comment:
I think we should only have one draw function with the given arguments.
We can later extend Animation and AnimatedSprite to use different draw mechanisms but in general that should be internal to AnimatedSprite and whoever calls the draw should not have to know how the AnimatedSprite does the drawing to pick the correct one..